pkg/activator/handler/enforce_length_handler_test.go

```diff
@@ -74,23 +74,24 @@ func TestEnforceMaxContentLengthHandler(t *testing.T) {
 			label:     "over without ContentLength",
 			maxUpload: len(payload) - 1,
 			request:   stream(payload),
-			status:    http.StatusOK,
-			response:  payload[0 : len(payload)-1],
+			status:    http.StatusRequestEntityTooLarge,
```

Comment:
So this is a non-backwards compatible change? :)
FWIW, I think this is right and I commented as such when the change was made, but oh well.